protocols.html

```diff
@@ -70,7 +71,8 @@ <h4>DIDComm Approach</h4>
                 formally documented in Aries RFC 0023</a>; only a summary is offered here. It has 3 steps:
```

Comment:
FYI - this should be updated to reflect the revision in Aries to have OutOfBand and DID Exchange as separate RFCs.

Reply:
Agreed. Done in 861172f.